src/mongo/mongoConnectionStrings.ts

```diff
@@ -63,23 +63,16 @@ export async function parseMongoConnectionString(connectionString: string): Prom
         }
     }
 
-    const serverConfig: Server | ReplSet | Mongos = mongoClient.db(testDb).serverConfig;
-    // Azure CosmosDB comes back as a ReplSet
-    if (serverConfig instanceof ReplSet) {
```

Comment:
All the mongo attached accounts are are not passing this condition and `host = (<any>serverConfig).host;` is returning undefined as `host` and `port` are not available at the top level of the object even for a single server.

Reply:
Is there an equivalent way to perform this check using the new SDK? I'm concerned that removing it could lead to problems

Reply:
Not really, with the new sdk the serverConfig is of type "any". 
![image](https://user-images.githubusercontent.com/17537419/101091702-6230ba00-356d-11eb-8089-34f6bcb968e7.png)

However, oddly the documentation here http://mongodb.github.io/node-mongodb-native/3.6/api/Db.html, doesn't show these objects as deprecated.

Reply:
Yeah it looks like those types still exist. Did you add the `@types` for the new SDK to package.json? I think excluding that may mess up the typing

Reply:
Whatever you figure out, I would prefer to wait for this change until after the release because of how often `parseMongoConnectionString` is used and because of the wide variety of mongo accounts that can be attached